src/index.ts

```diff
@@ -1,7 +1,9 @@
 import "./jecs"
-import "./pebble"
 import "./t"
 import "./vide"
+if (game.GetService("RunService").IsClient()) {
+    import("./pebble")
+}
```

Comment:
I actually don't think we need any of these importing LOC after all. Maybe test removing them first as that might fix the issue?

Reply:
Removing the importing LOC breaks the imports for `src/jabby/jecs`, `src/jabby/pebble`,  `src/jabby/t`,  and `src/jabby/vide`. I think this is due to how the TS variable gets compiled. Right now those imports compile to `local TS = _G[script]`, but changing them to `local TS = require(game:GetService("ReplicatedStorage"):WaitForChild("rbxts_include"):WaitForChild("RuntimeLib"))` fixes it. This LOC is how the TS variable is imported in my project, and I am not really sure why it is being compiled differently here.

Reply:
Yeah, I think these imports need to stay since it is being compiled as package. Looks like compiling as a game or model compiles the `TS` variable the different way I mentioned.

Reply:
Ok then let’s remove the dynamic import here as we don’t want a promise/async import. Let’s deal with the behaviour inside pebble

Reply:
Ok, I'll just change it to a normal import then.
Not sure how you want to handle it, but `pebble/components/display/widget` seems to be the only problem because of `Players.LocalPlayer.PlayerGui`. Could probably just only run that vide.mount() function if `RunService:IsClient()` is true.